lib/pathname.rb

```diff
@@ -570,6 +843,280 @@ def relative_path_from(base_directory)
   end
 end
 
+class Pathname    # * IO *
+  #
+  # #each_line iterates over the line in the file.  It yields a String object
+  # for each line.
+  #
+  # This method has existed since 1.8.1.
+  #
+  def each_line(...) # :yield: line
+    IO.foreach(@path, ...)
+  end
+
+  # See <tt>IO.read</tt>.  Returns all data from the file, or the first +N+ bytes
+  # if specified.
+  def read(*args) IO.read(@path, *args) end
```

Comment:
All these methods should probably use `...` consistently? `IO.read` can take keyword args now, but overall when delegating it's best if it keeps working regardless of how the delegated method signature evolve in the future.

Reply:
Yes, completely agreed, and in fact some of these caused bugs in the past, I will push a commit changing to `...` everywhere.